types/index.d.ts

```diff
@@ -0,0 +1,141 @@
+// TypeScript Version: 3.0
```

Comment:
https://devblogs.microsoft.com/typescript/announcing-typescript-3-4/#type-checking-for-globalthis

```suggestion
// TypeScript Version: 3.4
```